src/beast/beast/asio/bind_handler.h

```diff
@@ -23,142 +23,119 @@
 #include <boost/asio/detail/handler_alloc_helpers.hpp>
 #include <boost/asio/detail/handler_cont_helpers.hpp>
 #include <boost/asio/detail/handler_invoke_helpers.hpp>
-
 #include <functional>
 #include <type_traits>
 #include <utility>
 
 namespace beast {
-namespace asio {
 
 namespace detail {
 
 /** Nullary handler that calls Handler with bound arguments.
-    The rebound handler provides the same io_service execution
+
+    The bound handler provides the same io_service execution
     guarantees as the original handler.
 */
-template <class DeducedHandler, class... Args>
+template<class DeducedHandler, class... Args>
 class bound_handler
 {
 private:
     using args_type = std::tuple <std::decay_t <Args>...>;
+    
```

Comment:
When a class is using lots of index sequences based on the classes template args, I wonder if a `using arg_seq = std::index_sequence_for<Args...>` here would help speed up compilation. This class only uses two, but I'll keep an eye out for other classes that may use more.